tests/preferences/conftest.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+"""Fixtures for unittesting the preferences submodule."""
+
+from __future__ import absolute_import, unicode_literals
+
+import datetime
+import os.path
+
+import pytest
+from gi.repository import Gtk
+
+from hamster_gtk.preferences import widgets
+
+
+# Instances
+
+@pytest.fixture
+def config_combo_box_text(request, combo_box_items):
+    """Return a ConfigComboBoxText instance."""
+    widget = widgets.ConfigComboBoxText()
+    for id, item in combo_box_items:
+        widget.append(id, item)
+    return widget
+
+
+@pytest.fixture
+def editable_file_chooser(request):
+    """Return a EditableFileChooser instance."""
+    return widgets.EditableFileChooser()
+
+
+@pytest.fixture
+def duration_entry(request, adjustment):
+    """Return a DurationEntry instance."""
+    spin = widgets.DurationEntry()
+    spin.set_adjustment(adjustment)
+    return spin
+
+
+@pytest.fixture
+def time_entry(request):
+    """Return a TimeEntry instance."""
+    return widgets.TimeEntry()
+
+
+# Data
+
```

Comment:
The global config contains a ``initial_config_parametrized`` fixture that is used for testing preferences only. This and its associated fixtures should probably be moved over here as well.